pkgs/by-name/ro/rose-pine-cursor/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rose-pine-cursor";
+  version = "1.1.0";
+
+  srcs = [
+  (fetchurl {
+    url = "https://github.com/rose-pine/cursor/releases/download/v${version}/BreezeX-RosePine-Linux.tar.xz";
+    hash = "sha256-szDVnOjg5GAgn2OKl853K3jZ5rVsz2PIpQ6dlBKJoa8=";
+  })
+  (fetchurl {
+    url = "https://github.com/rose-pine/cursor/releases/download/v${version}/BreezeX-RosePineDawn-Linux.tar.xz";
+    hash = "sha256-hanfwx9ooT1TbmcgCr63KVYwC1OIzTwjmxzi4Zjcrdg=";
+  })
+  ];
+
+  sourceRoot = ".";
+
+  installPhase = ''
+  mkdir -p $out/share/icons
+  ls -la
+  cp -r BreezeX-RosePine-Linux $out/share/icons/
+  cp -r BreezeX-RosePineDawn-Linux $out/share/icons/
+  '';
```

`cp -R` does the same thing as -r while being more portable
```suggestion
  installPhase = ''
    mkdir -p $out/share/icons
    cp -R BreezeX-RosePine-Linux $out/share/icons/
    cp -R BreezeX-RosePineDawn-Linux $out/share/icons/
  '';
```